docs/site/migration/boot-scripts.md

```diff
@@ -6,8 +6,249 @@ sidebar: lb4_sidebar
 permalink: /doc/en/lb4/migration-boot-scripts.html
 ---
 
-{% include note.html content="
-This is a placeholder page, the task of adding content is tracked by the
-following GitHub issue:
-[loopback-next#3957](https://github.com/strongloop/loopback-next/issues/3957)
-" %}
+In LoopBack 3, predefined boot scripts are organized in the `/server/boot`
+directory, and are executed right before the server starts to perform some
+custom application initialization. The same functionality can also be achieved
+in LoopBack 4 application by adding observers.
+
+The LoopBack 3 boot script is mapped to LoopBack 4 observer's `start` function.
+And moreover, besides participating in an application's start phase, an observer
+also allows you to perform logic in the stop phase.
+
+## Migrating Boot Scripts
+
+### LoopBack 3 Boot Script
+
+LoopBack 3 has different signatures for synchronized and asynchronized boot
```

Comment:
AFAIK, we use the terms "synchronous" and "asynchronous", not synchronized and asynchronized. See e.g. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function.

```suggestion
LoopBack 3 has different signatures for synchronous and asynchronous boot
```

Please update all new content accordingly.